src/Components/Articles.jsx

```diff
@@ -0,0 +1,30 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import ArticleCard from "./ArticleCard";
+import ChangePage from "./SwitchPage";
+
+export default function Articles() {
+  const [articleList, setArticleList] = useState([]);
+  const [page, setPage] = useState(1)
+
+  useEffect(() => {
+    axios
+      .get(`https://northcoders-news-03ck.onrender.com/api/articles?p=${page}`)
+      .then(({ data: { articles } }) => {
+        setArticleList(articles);
+        window.scrollTo(0, 0)
+      });
+  }, [page]);
+
+  return (
+    <div>
+      <h1>Articles</h1>
+      <ChangePage setPage={setPage} page={page}/>
+      <ul>{articleList.map((article) => {
+        return <ArticleCard key={article.article_id} article={article}/>
+      })}</ul>
+      <ChangePage setPage={setPage} page={page}/>
+      <p>Page: {page}</p>
+    </div>
+  );
+}
```

Comment:
A loading state is needed here so the user has some feedback while the page is loading 🙂